web/packages/build/package.json

```diff
@@ -69,15 +69,15 @@
     "jsdom-testing-mocks": "^1.13.0",
     "msw": "^0.47.4",
     "node-gyp": "^10.1.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
```

Comment:
@avatus Are we going to backport these updates, do we have that codified anywhere?

I feel like they should be backported to at least the most recent release branch. Otherwise for the duration of v16 we'll keep releasing new features that were developed on a newer version of React that is being used in "production".

Reply:
It isn't codified. This process mostly followed the backend team's process which is "only backport security patches" but I'm sure there are some case by case basis choices going on there. 

I'm happy to deviate and say we, the frontend team, can backport dep changes to the release branch if the rest of the team is.